apps/meteor/client/views/composer/EmojiPicker/EmojiElement.tsx

```diff
@@ -1,24 +1,35 @@
-import { Box, IconButton } from '@rocket.chat/fuselage';
+import { css } from '@rocket.chat/css-in-js';
+import { IconButton } from '@rocket.chat/fuselage';
 import type { MouseEvent, AllHTMLAttributes } from 'react';
 import React, { memo } from 'react';
 
 import type { EmojiItem } from '../../../../app/emoji/client';
+import Emoji from '../../../components/Emoji';
 import { usePreviewEmoji } from '../../../contexts/EmojiPickerContext';
 
-type EmojiElementProps = EmojiItem & { onClick: (e: MouseEvent<HTMLElement>) => void } & Omit<AllHTMLAttributes<HTMLButtonElement>, 'is'>;
+type EmojiElementProps = EmojiItem & { small?: boolean; onClick: (e: MouseEvent<HTMLElement>) => void } & Omit<
+		AllHTMLAttributes<HTMLButtonElement>,
+		'is'
+	>;
 
-const EmojiElement = ({ emoji, image, onClick, ...props }: EmojiElementProps) => {
+const EmojiElement = ({ emoji, image, emojiHandle, onClick, small = false, ...props }: EmojiElementProps) => {
 	const { handlePreview, handleRemovePreview } = usePreviewEmoji();
-	if (!image) {
-		return null;
-	}
 
-	const emojiElement = <Box dangerouslySetInnerHTML={{ __html: image }} />;
+	const emojiSmallClass = css`
+		> span {
```

Comment:
span is a bit broad selector, dont you have anything like .emoji or something else?